corehq/apps/users/templates/users/mobile_workers.html

```diff
@@ -104,6 +104,12 @@
                 {% trans "Edit User Fields" %}
               </a>
             {% endif %}
+            {% if request|toggle_enabled:"CLEAR_MOBILE_WORKER_DATA" %}
+              <a id="clear-data" class="btn btn-default" href="{% url "clear_commcare_users" domain %}">
```

Comment:
```suggestion
              <a id="clear-data" class="btn btn-danger" href="{% url "clear_commcare_users" domain %}">
```
Since this is fundamentally a delete operation, this button should be styled as `btn-danger`.